save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/SandboxFileInfo.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.domain
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Class that contains metadata of Sandbox files
+ *
+ * @property sizeBytes size in bytes
+ * @property name name of a file
+ */
+@Serializable
+data class SandboxFileInfo(
```

Comment:
Will we not make a File table for the sandbox?

Reply:
should do, will create an issue for it

Reply:
https://github.com/saveourtool/save-cloud/issues/1718